cloudformation/editorial-viewer.json

```diff
@@ -43,6 +48,10 @@
       "Description": "Github team name, used for giving ssh access to members of the team.",
       "Type": "String",
       "Default": "Editorial-Tools-SSHAccess"
+    },
+    "ImageId": {
```

Comment:
minor: Default name for the AMI id parameter in Riff-Raff is `AMI`. Calling it this way might save you one line in your `riff-raff.yaml` or `deploy.json`